kong.config.sh

```diff
@@ -29,39 +29,27 @@ PAYLOAD
     --header "Content-Type: application/json" \
     -d @- ) <<PAYLOAD
 {
-    "name": "metric",
-    "uris": "/metric",
-    "strip_uri": true,
-    "upstream_url": "http://orion:1026"
-}
-PAYLOAD
-authConfig "metric"
-
-
-(curl -o /dev/null $kong/apis -s -S -X POST \
-    --header "Content-Type: application/json" \
-    -d @- ) <<PAYLOAD
-{
-    "name": "template",
-    "uris": "/template",
+    "name": "data-broker",
+    "uris": [/device/(.*)/latest", "/subscription"],
```

Comment:
This /device/*/latest endpoint will hold all current device attribute values, is that correct?

Reply:
precisely - an argument could be made that history should be the one responsible for that, but to implement more complicated subscription patterns data-broker needs this kind of information too.